attic/security/retorts.py

```diff
@@ -0,0 +1,221 @@
+from base64 import (
```

Comment:
This file should not be in the attic since it is still present in `src`. Just deleting the unused artifacts from the copy in `src` should suffice.